sdk/quoting/public/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We can remove this file in the template and here, I missed that while polishing!